src/Document/Query.php

```diff
@@ -88,7 +88,7 @@ public static function cssToXpath($path)
 
         // Arbitrary attribute value contains whitespace
         $path = preg_replace_callback(
-            '/\[\S+["\'](.+)["\']\]/',
+            '/\[\S+["\']([\w\s]+)["\']\]/',
```

Comment:
Hm... what about `.+?` (non greedy)

Reply:
Ok, I had a look on it, and `.+?` doesn't work because `\S+` is greedy. So I would suggest then:
```
/\[\S+?["\'](.+?)["\']\]/
```
or with modifier `U` (see: http://php.net/manual/en/reference.pcre.pattern.modifiers.php):
```
'/\[\S+["\'](.+)["\']\]/U'
```